lib/lutaml/uml/class.rb

```diff
@@ -9,7 +9,8 @@ module Uml
     class Class < Classifier
       class UknownMemberTypeError < StandardError; end
```

Comment:
Can we differentiate between Class::UknownMemberTypeError and Document::UknownMemberTypeError with different names?

Reply:
Sure, will do. I have deleted it for now though, we don't need it for now.